app/javascript/components/recipes/show/base-instruction-step.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="flex flex-row items-start py-2 w-auto h-auto flex-grow-0">
+    <div class="w-6 h-6 p-3 flex items-center justify-center order-0 flex-grow-0 mr-3 rounded-full bg-gray-200 font-sans font-normal not-italic text-black text-xs">
```

Comment:
Cuál es la diferencia entre font-normal y font-sans? Y es necesario el not-italic?

Reply:
font-normal se refiere al "weight" como si es mas bold o mas light, y font-sans se refiere a una tipografía de tipo sans. El not-italic cache que no es necesario así que los estoy borrando